plugins/ILibPlugin/index.js

```diff
@@ -169,10 +169,14 @@ class ILibPlugin {
 				if (pkgName.indexOf('@enact') === 0) {
 					this.options.create = false;
 				}
-				if (pkgName === '@enact/i18n') {
-					this.options.ilib = 'ilib';
-				} else {
-					this.options.ilib = packageSearch(process.cwd(), '@enact/i18n/ilib');
+				// look for ilib as a root-level node_module package location
+				this.options.ilib = packageSearch(process.cwd(), 'ilib');
+				if (!this.options.ilib) {
+					// if not found, look for a nested ilib dependency within @enact/i18n
+					this.options.ilib = packageSearch(
+						process.cwd(),
+						path.join('@enact', 'i18n', 'node_modules', 'ilib')
```

Comment:
I don't think this path would be valid with `ilib` as a peer dependency. Should this resolve to `@enact/i18n/ilib` for backwards compatibility instead?